### DISCUSSION TITLE
why does it look like this?

### DISCUSSION THREAD

**Discussion by Sonicissocool** [anthony1x6000/ROBLOX2016stylus#36] [Q&A]
the Roblox homepage shouldn't look like a fusion between modern and 2016 Roblox?/nm

![image](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/137438818/2aadc3ad-365c-4a68-a82a-3e9a815103b9)

**Comment by Cordyte** [comment]
wow, i think its some sort of bug?? well its the old roblox ui but dark mode, you can probaly fix it by just refreshing the website..

**Comment by anthony1x6000** [comment]
Unfortunately, there is no support for dark mode. You will have to change the "Theme" located within Settings > Account Info to "Light"
![image](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/33004321/6e96b7b5-71b0-4ebf-a4d0-fc53a70d80ba)
